### PR TITLE
Promote aws-account-operator e2b823c

### PR DESCRIPTION
**Pull request by jharrington22**
This PR promotes the latest version of the aws-account-operator to production.

### CODE REVIEW

**Comment by jharrington22** [comment]
@jmelis this PR is good to merge if @jhernand and @nimrodshn confirm once they test staging.

**Review by jmelis — APPROVED** [review]
LGTM! but let's wait for confirmation that it's working properly

**Comment by jharrington22** [comment]
Closed PR as there have been changes.

**Pull request closed without merging**
